.gitignore

```diff
@@ -1,5 +1,33 @@
 *.py[cod]
 
+# Compiled Object files
+*.slo
+*.lo
+*.o
+*.obj
+
+# Compiled Dynamic libraries
+*.so
+*.dylib
+
+# Compiled Static libraries
+*.lai
+*.la
+*.a
+
+# User files
+*.user
+*.suo
+
+# Compiled executable
+!src/*.exe
+
+# Dynamic link library
+!src/*.dll
+
+# KA Lite source folder
+ka-lite/
```

Comment:
this shouldn't exist! :)

Reply:
Thanks, @indirectlylit for reviewing, I will remove that today.